documentation/installation.md

```diff
@@ -49,16 +60,26 @@ command:
 
     vagrant ssh
 
-To check if everything is set up properly you can run ``make test.unit`` inside
+Inside the virtual machine, if you are setting up a development environment,
+set HUEY_ALWAYS_EAGER to false in /home/vagrant/opencraft/.env.
+
+Inside the virtual machine, create a super user which will be used to login
+in OCIM:
+
+    make manage createsuperuser
+
+To check if everything is set up properly, you can run ``make test.unit`` inside
 your new environment.
 
 Vagrant will set up a VirtualBox share mapping your local development directory
 to `/vagrant` inside the virtual machine. Any changes you make locally will be
 reflected inside the virtual machine automatically.
 
 Vagrant will map port 5000 inside the virtual machine to port 5000 on the host.
-Once you have set everything up you will be able to access the development
-server at http://localhost:5000/ using your web browser.
+Once you have set everything up, you will be able to access the home webpage of
+the development server at http://localhost:5000/ using your web browser. Login at
+http://localhost:5000/admin with the previously created superuser and go back to
+the home page in order to access OCIM.
```

Comment:
@alanoe, why does the user have to log in via the Django admin interface? The regular login form at `/registration/login/` works just fine.

```suggestion
the home page in order to access Ocim.
```

Reply:
> @alanoe, why does the user have to log in via the Django admin interface? The regular login form at `/registration/login/` works just fine.

I thought that the superuser created by "make manage superuser" was only valid for the Django admin UI. I'll change the doc then.